### PR TITLE
Dogfood setup-regal action

### PR DESCRIPTION
**Pull request by charlieegan3**
Uses https://github.com/StyraInc/setup-regal

### CODE REVIEW

File: .github/workflows/build.yaml [hunk 1]
```diff
@@ -24,3 +24,19 @@ jobs:
       - uses: golangci/golangci-lint-action@v3
         with:
           version: v1.52.2
+  # lint-bundle is here to dogfood the setup-regal action from https://github.com/StyraInc/setup-regal
+  lint-bundle:
+    name: Lint Bundle
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check out code
+        uses: actions/checkout@v3
```
Comment (anderseknert) [inline .github/workflows/build.yaml:33]: Might as well make this v4 right away, as it's used elsewhere in this file

File: bundle/regal/ast.rego [hunk 2]
```diff
@@ -1,9 +1,6 @@
 package regal.ast
 
-import future.keywords.contains
-import future.keywords.every
-import future.keywords.if
-import future.keywords.in
+import future.keywords
```
Comment (anderseknert) [inline bundle/regal/ast.rego:3]: This should be reverted, I guess?

File: .github/workflows/build.yaml (hunk 1 above)
Comment (charlieegan3) [reply .github/workflows/build.yaml]: 👍 fixed

**Review by anderseknert — APPROVED** [review]

**Comment by anderseknert** [comment]
Neato!

**Pull request merged**